test/db/formats/coff

```diff
@@ -58,26 +58,25 @@ NAME=tiny coff
 FILE=bins/coff/coff.obj
 CMDS=om;is;ir
 EXPECT=<<EOF
- 3 fd: 3 +0x00000000 0x00000000 - 0x00000144 r-x 
- 2 fd: 3 +0x00000064 0x00000064 - 0x0000008a r-x fmap..text_0
- 1 fd: 3 +0x0000008b 0x0000008b - 0x000000a6 r-- fmap..data_1
+ 2 fd: 3 +0x00000064 0x00000000 - 0x00000026 r-x fmap..text_0
+ 1 fd: 3 +0x0000008b 0x00000030 - 0x0000004b r-- fmap..data_1
 [Symbols]
 
 nth paddr       vaddr              bind   type size lib name
 ------------------------------------------------------------
 0    ---------- 0xffffffffffffffff NONE   UNK  4        imp.MessageBoxA
 0    ---------- 0xffffffffffffffff NONE   UNK  4        imp.ExitProcess
-0    0x00000064 0x00000064         LOCAL  SECT 4        .text
-0    0x00000064 0x00000064         GLOBAL FUNC 4        main
-0    0x0000008b 0x0000008b         LOCAL  SECT 4        .data
+0    0x00000064 0x00000000         LOCAL  SECT 4        .text
+0    0x00000064 0x00000000         GLOBAL FUNC 4        main
+0    0x0000008b 0x00000030         LOCAL  SECT 4        .data
 [Relocations]
 
 vaddr      paddr      type   name
 ---------------------------------
-0x0000006d 0x0000006d ADD_32 .data
-0x00000074 0x00000074 ADD_32 .data
-0x00000080 0x00000080 ADD_32 MessageBoxA
-0x00000087 0x00000087 ADD_32 ExitProcess
+0x00000009 0x0000006d ADD_32 .data
```

Comment:
these offsets look wrong to me but I didnt looked in depth. can you confirm?

Reply:
`LEA`
0x06 + 0x03 = 0x09
0x0d + 0x03 = 0x10
`CALL`
0x1b + 0x01 = 0x1c
0x22 + 0x01 = 0x23
```asm
0x00000000         41b900000000  mov r9d, 0
0x00000006       4c8d0523000000  lea r8, [rip + 0x23]
0x0000000d       488d152b000000  lea rdx, [rip + 0x2b]
0x00000014       48c7c100000000  mov rcx, 0
0x0000001b           e830000000  call 0x50
0x00000020                 89c1  mov ecx, eax
0x00000022           e831000000  call 0x58
```